seed/seed.go

```diff
@@ -222,6 +233,11 @@ type Copier interface {
 	// on this Seed with a different mode, then that metadata will be
 	// overwritten by the metadata for all modes.
 	Copy(seedDir string, label string, tm timings.Measurer) error
+	// OptionalContainers returns the set of snaps and components that are
+	// consider optional in the seed's model, but are available in the seed and
+	// can be copied to a new seed location. Use this in conjunction with
+	// Copier.Copy specific optional snaps and components.
```

Comment:
is there something missing in the last sentence?  "to copy specific ..." ? "to choose ..."?